front-end/src/ui-components/InfoBox.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactNode, useState } from 'react';
+import styled from '@xstyled/styled-components';
+import { MdClose } from 'react-icons/md';
+
+interface Props{
+	children?: ReactNode
+	className?: string
+	content?: string
+	dismissable?: boolean
+	name: string
+	title: string
+}
+
+const InfoBox = ({ children, className, content, dismissable, name, title }: Props) => {
+	const infoBoxVisible = localStorage.getItem(name);
```

Comment:
Please don't forget my first comment, adding `Visible` to the name help clarifying what this does.
```suggestion
	const localStorageName = name + 'Visible'
	const infoBoxVisible = localStorage.getItem(localStorageName);
```